pom.xml

```
@@ -79,30 +117,30 @@
       <version>25.0-jre</version>
       <!--<scope>test</scope>-->
     </dependency>
-    <!--<dependency>-->
-    <!--<groupId>org.apache.httpcomponents</groupId>-->
-    <!--<artifactId>httpclient</artifactId>-->
-    <!--<version>4.5.3</version>-->
-    <!--<scope>test</scope>-->
-    <!--</dependency>-->
-    <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>fluent-hc</artifactId>
-      <version>4.5.3</version>
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.eclipse.jgit</groupId>
       <artifactId>org.eclipse.jgit</artifactId>
-      <version>4.7.0.201704051617-r</version>
+      <version>5.12.0.202106070339-r</version>
```

The most recent version supported by the git client plugin is:

Since this is a test dependency, you can leave it as the older version if you wish.

Can't make it depend on JGit 6.0 because that does not compile with Java 8.
```suggestion
      <version>5.13.0.202109080827-r</version>
```